baby-gru/src/components/BabyGruMoleculeCard.js

```diff
@@ -14,6 +14,77 @@ export const BabyGruMoleculeCard = (props) => {
     const [isCollapsed, setIsCollapsed] = useState(!props.defaultExpandDisplayCards);
     const [popoverIsShown, setPopoverIsShown] = useState(false)
     const [isVisible, setIsVisible] = useState(true)
+    const [bondWidth, setBondWidth] = useState(0.1)
+    const [atomRadiusBondRatio, setAtomRadiusBondRatio] = useState(1.5)
+    const [bondSmoothness, setBondSmoothness] = useState(1)
+
+    const isDarkBackground = (backgroundColor) => {
+        const bright_y = backgroundColor[0] * 0.299 + backgroundColor[1] * 0.587 + backgroundColor[2] * 0.114
+        if (bright_y >= 0.5) {
+            return false
+        }
+        return true
+    }
```

Comment:
@stuartjamesmcnicholas the logic used to determine whether the background is dark or not is the same as the one used in `mgWebGL` to determine whether the atom labels should be white or black. I think this is OK?